blivet/gpt.py

```diff
@@ -0,0 +1,228 @@
+        "s390x": uuid.UUID("8a4f5770-50aa-4ed3-874a-99b710db6fea"),
+        "tilegx": uuid.UUID("55497029-c7c1-44cc-aa39-815ed1558630"),
+        "x86_64": uuid.UUID("8484680c-9521-48c6-9c11-b0720656f69e"),
+    },
+    GPT_VOL_ARCH_USR_VERITY: {
+        "alpha": uuid.UUID("8cce0d25-c0d0-4a44-bd87-46331bf1df67"),
+        "arc": uuid.UUID("fca0598c-d880-4591-8c16-4eda05c7347c"),
+        "arm": uuid.UUID("c215d751-7bcd-4649-be90-6627490a4c05"),
+        "arm64": uuid.UUID("6e11a4e7-fbca-4ded-b9e9-e1a512bb664e"),
+        "i686": uuid.UUID("8f461b0d-14ee-4e81-9aa9-049b6fb97abd"),
+        "ia64": uuid.UUID("6a491e03-3be7-4545-8e38-83320e0ea880"),
+        "loongarch64": uuid.UUID("f46b2c26-59ae-48f0-9106-c50ed47f673d"),
+        "mips64el": uuid.UUID("3c3d61fe-b5f3-414d-bb71-8739a694a4ef"),
+        "mipsel": uuid.UUID("46b98d8d-b55c-4e8f-aab3-37fca7f80752"),
+        "parisc": uuid.UUID("5843d618-ec37-48d7-9f12-cea8e08768b2"),
+        "ppc": uuid.UUID("df765d00-270e-49e5-bc75-f47bb2118b09"),
+        "ppc64": uuid.UUID("bdb528a5-a259-475f-a87d-da53fa736a07"),
+        "ppc64el": uuid.UUID("ee2b9983-21e8-4153-86d9-b6901a54d1ce"),
+        "riscv32": uuid.UUID("cb1ee4e3-8cd0-4136-a0a4-aa61a32e8730"),
+        "riscv64": uuid.UUID("8f1056be-9b05-47c4-81d6-be53128e5b54"),
+        "s390": uuid.UUID("b663c618-e7bc-4d6d-90aa-11b756bb1797"),
+        "s390x": uuid.UUID("31741cc4-1a2a-4111-a581-e00b447d2d06"),
+        "tilegx": uuid.UUID("2fb4bf56-07fa-42da-8132-6b139f2026ae"),
+        "x86_64": uuid.UUID("77ff5f63-e7b6-4633-acf4-1565b864c0e6"),
+    },
+    GPT_VOL_ARCH_USR_VERITY_SIG: {
+        "alpha": uuid.UUID("5c6e1c76-076a-457a-a0fe-f3b4cd21ce6e"),
+        "arc": uuid.UUID("94f9a9a1-9971-427a-a400-50cb297f0f35"),
+        "arm": uuid.UUID("d7ff812f-37d1-4902-a810-d76ba57b975a"),
+        "arm64": uuid.UUID("c23ce4ff-44bd-4b00-b2d4-b41b3419e02a"),
+        "i686": uuid.UUID("974a71c0-de41-43c3-be5d-5c5ccd1ad2c0"),
+        "ia64": uuid.UUID("8de58bc2-2a43-460d-b14e-a76e4a17b47f"),
+        "loongarch64": uuid.UUID("b024f315-d330-444c-8461-44bbde524e99"),
+        "mips64el": uuid.UUID("f2c2c7ee-adcc-4351-b5c6-ee9816b66e16"),
+        "mipsel": uuid.UUID("3e23ca0b-a4bc-4b4e-8087-5ab6a26aa8a9"),
+        "parisc": uuid.UUID("450dd7d1-3224-45ec-9cf2-a43a346d71ee"),
+        "ppc": uuid.UUID("7007891d-d371-4a80-86a4-5cb875b9302e"),
+        "ppc64": uuid.UUID("0b888863-d7f8-4d9e-9766-239fce4d58af"),
+        "ppc64el": uuid.UUID("c8bfbd1e-268e-4521-8bba-bf314c399557"),
+        "riscv32": uuid.UUID("c3836a13-3137-45ba-b583-b16c50fe5eb4"),
+        "riscv64": uuid.UUID("d2f9000a-7a18-453f-b5cd-4d32f77a7b32"),
+        "s390": uuid.UUID("17440e4f-a8d0-467f-a46e-3912ae6ef2c5"),
+        "s390x": uuid.UUID("3f324816-667b-46ae-86ee-9b0c0c6c11b4"),
+        "tilegx": uuid.UUID("4ede75e2-6ccc-4cc8-b9c7-70334b087510"),
+        "x86_64": uuid.UUID("e7bb33fb-06cf-4e81-8273-e543b413e2e2"),
```

Comment:
This is more a note to myself: we should move these definitions to libblockdev in the future (possibly using the GUID list provided by libfdisk). We had similar discussion recently about these definitions in UDisks https://github.com/storaged-project/udisks/pull/1032 and it makes sense to have these in one place if possible. (But that's out of scope for this PR.)

Reply:
Yeah, it would make sense to have them in a place that can maximise sharing.